pyproject.toml

```diff
@@ -0,0 +1,21 @@
+[build-system]
+requires = ['setuptools', 'wheel']
+
+[tool.black]
+line-length = 100
```

Comment:
I was using 120 limit

Reply:
Okay, I bumped that up to 120 and reformatted. Thanks for reviewing!